lib/views/notePageView.js

```diff
@@ -6,7 +6,7 @@
         ${NavBar().render()}
         ${Note().render()}
         <div>
-          <button id="edit" onclick="editNote()">Edit Note</button>
+          <button id="edit" onclick="NotesEditPageView().render()">Edit Note</button>
```

Comment:
this i don't would work
the onClick event handler needs to point to a method on the app.js

```suggestion
          <button id="edit" onclick="editNotePage(index)">Edit Note</button>
```